products/junit.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Junit
+addedAt: 2025-11-22
+category: framework
+tags: java-runtime
+iconSlug: junit5
+permalink: /junit
+
+releasePolicyLink: https://github.com/junit-team/junit-framework/?tab=security-ov-file#security-policy
+changelogTemplate: https://github.com/junit-team/junit-framework/releases/tag/v__LATEST__
+
+identifiers:
+  - repology: junit
+  - purl: pkg:github/junit-team/junit-framework
+
+auto:
+  methods:
+    - git: https://github.com/junit-team/junit-framework.git
+
+releases:
+  - releaseCycle: "6.0"
```

Comment:
Based on https://github.com/junit-team/junit-framework/commits/main/SECURITY.md history, it looks like the release cycle should be on the major, not the minor. I looked at multiple commits and it looks like once a minor is released, the previous minor is immediately EOL.